### DISCUSSION TITLE
Yi Pro 2K Home (ZFUS) No WiFi (Didn't run App)

### DISCUSSION THREAD

**Discussion by tpoliquin** [roleoroleo/yi-hack-Allwinner-v2#970] [Q&A]
I'm new to Yi-Hack .. I followed the install instructions (y623 0.3.3, and homever.txt -> 12.0.51.01_202303091901).

I didn't run the app .. took it out of the box connected it to their usb supply with their cable and did the install. Nice blue flashing LED but no Wifi. I did find a note on how to install without the app being run but it was deprecated ... It appears that that was rolled into 0.3.3 ..
I ran it anyway .. and still nada. I tried both using the standard config_wifi.cfg.ori and a custom one .. same problem.
I tried to find it with my openwrt router, nmcli and nmap .. nada. I put some echo debugs in config_wifi.sh (to a log file, since I don't know it's IP) and things appear to be running without errors, just no connectivity.
I must be doing something stupid.
I even tried connecting the USB to my laptop .. hoping I'd get a tty .. nope .. It sees it but complains "Device not responding to setup address."
I've been looking for awhile for a nice Linux inexpensive camera and this project seems awesome if I can just get over this hump!
Help appreciated.
Tom

**Comment by roleoroleo** [comment]
There are two different ways to connect the cam with your wifi.
1 - Running the app before applying the hack (or after the hack but removing the sd card).
2 - Following the readme using configure_wifi.sh (renaming configure_wifi,.cfg.ori to configure_wifi.cfg and editing it with your credentials).

The 1st procedure must work. If it doesn't work, probably there is a hardware problem with the wifi card.

**Reply by tpoliquin** [reply]
Thanks for the reply! 

I just found the problem .. Credit me with an error!
I was stupidly using the wrong SSID (even though I
checked it N times) ..

The camera is working great! Even 'on a dark and stormy night'
where I live the image is excellent.

Thanks for all the effort
![wuuk_night01](https://github.com/user-attachments/assets/57de3655-b6d1-4bf9-b96a-17c4b612a6b0)
in  making this camera work ..
I'm ordering a few more (great cheap Christmas presents -:) )

Tom
